### ISSUE TITLE
Use oauth with jira

### ISSUE THREAD

**Issue by cweiske** [kraiz/hamster-bridge#23]
Instead of saving the jira password in the config file, hamster-bridge should obtain and use an oauth token.

**Comment by omarkohl** [comment]
@cweiske I agree that oauth token is the way to go but fwiw: Since version 0.6.0 of hamster-bridge passwords are no longer stored in the config file (by default) and have to be entered interactively every time you start the application.
